TODO.Rmd

```diff
@@ -0,0 +1,43 @@
+---
+title: "TODO"
+author: "Zoe Meers"
+date: "14/07/2018"
+output: html_document
+---
+
+# TODO
+## Theme 
+
+We might need to make our own theme...
+I think theme_void() + theme(legend.postion = 'none') might work... Thoughts?
```

Comment:
Hmm, put the black part in the center? Let's try it and see what it looks like! Just switch around the layers -- new_layers plots first in this case.

Re: theme... I think we should take a look at https://github.com/hrbrmstr/hrbrthemes. There are a few interesting things here. I like that the user can just write TRUE/FALSE for legends or axes or whatever instead of `theme(legend.position='none')`. So if someone wants to draw party labels instead of using a legend, they can write `theme_ggparliament(legend=FALSE)`  or vice-versa if they want the legend. 

Moreover, if someone writes `theme_ggparliament(legend=TRUE)` we can set the legend so that it plots on the bottom. That may screw up the opposing bench style, though. Do you think we should we flip it around so that there is a bench on top and one below? Or I suppose it can switch the type, then plot to the side... 

My point is: I think we should have an opinionated theme, otherwise the user can just create their own.